pymess/backend/emails/mandrill.py

```diff
@@ -52,7 +52,7 @@ class MandrillEmailBackend(EmailBackend):
         'PRESERVE_RECIPIENTS': False,
         'VIEW_CONTENT_LINK': True,
         'ASYNC': False,
-        'TIMEOUT': 5,  # 5s
```

Comment:
this is default configuration you can change it in the project settings please do it

Reply:
ok, makes sense. I am closing this then